app/controllers/public_submissions_controller.rb

```diff
@@ -0,0 +1,88 @@
+# encoding: utf-8
+
+class PublicSubmissionsController < ApplicationController
+  attr_reader :submission, :authorship
+
+  def new
+    @submission = Submission.new
+    submission.article_revisions.build(version: 1)
```

Comment:
Hmm, ciekaw jestem gdzie Pani o tym słyszała ;-) Tzn. nie żeby to rozwiązanie było jakieś bardzo nieładne, ale zdecydowanie nie jest standardowe. Jeśli nie używa się go w całej aplikacji (jak tutaj), to wprowadza tylko dodatkowy poziom skomplikowania). Ale zostawmy to tak.